pkgs/development/tools/profiling/oprofile/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, binutils, popt, zlib, pkgconfig
+{ stdenv, fetchurl, binutils, popt, zlib, pkgconfig, linuxPackages
```

Comment:
Well, it's my first try to change anything in nixpkgs.
Actually I don't know how to pass only kernelHeaders here. Do you have any suggestion?